content/using/running/minio.md

```diff
@@ -0,0 +1,176 @@
++++
+title = "Self-hosting S3 Storage"
+description = "How to set up and self-host MinIO S3 storage"
+template = "doc.html"
+weight = 2
+[extra]
+hidetitle = "true"
++++
+
+Adding [S3](https://aws.amazon.com/s3/) storage to Urbit unlocks some great new features, such as the ability to post media to chats and upload custom avatars. This is a guide to self-hosting [MinIO](https://min.io), an S3 compatible block storage solution.
+
+You can read more about S3 in [Configuring S3 Storage](/using/os/s3).
+
+Cloud providers offer off-the-shelf S3 solutions that you can get started with almost immediately. However, if you do not wish to trust them with your uploaded files, you can self-host an S3 solution.
+
+This process requires a working knowledge of the Linux command line and web technologies such as DNS and TLS.
```

Comment:
Can't this be done on the Mac as well? https://backup.ninja/news/how-build-your-own-object-storage-using-minio 

The grant proposal doesn't say anything about being particular to Linux, so we should have Mac OS instructions as well.